Use annotations in floating interdomain examples

d-uzlov opened the pull request:
<!--- Put an `x` in all the boxes that this PR applies -->

## Description
<!--- Provide a general summary of your changes in the Title above -->

Change examples to use webhook+annotations instead of patching bare nsc-kernel.

## Issue link
<!--- Please link to the issue here. -->

https://github.com/networkservicemesh/deployments-k8s/issues/7673

## How Has This Been Tested?
<!--- Provide information on how these changes are testing -->
- [ ] Added unit testing to cover
- [x] Tested manually
- [x] Tested by integration testing
- [ ] Have not tested

<!--- Add additional comments about testing if needed. -->

## Types of changes
<!--- What types of changes does your code introduce -->
- [ ] Bug fix
- [ ] New functionallity
- [ ] Documentation
- [x] Refactoring
- [ ] CI

denis-tingaikin reviewed (commented):
Looks good.

Only one note, can we exclude 'fi' from ns naming?

Currently, we're using the name of the test as a namespace.

So `ns-kernel2wireguard2kernel-fi-cluster-2` is going to be `ns-floating-ns-kernel2wireguard2kernel

denis-tingaikin commented:
Could you also rebase the branch?

d-uzlov commented:
@denis-tingaikin done.
I renamed namespaces, updated use-case folder names, and renamed `floating_interdomain` folder into `multicluster`, as we discussed.

denis-tingaikin commented:
@d-uzlov Nice! Seems like we need one more rebase

d-uzlov commented:
@denis-tingaikin, done
